command/helper/helper.go

```diff
@@ -476,13 +479,25 @@ func ReadConfig(baseCommand string, args []string) (*Config, error) {
 		return nil, err
 	}
 
+	if cliConfig.Network.MaxPeers != -1 {
```

Comment:
Where do you explicitly check that MaxInboundPeers and MaxOutboundPeers are set correctly?

Imagine I put that MaxInboundPeers is 5 and MaxOutboundPeers to be -1, where do you guard against that?